tests/helpers/test_config_entry_oauth2_flow.py

```diff
@@ -69,6 +69,11 @@ def domain(self) -> str:
         """Domain that is providing the implementation."""
         return "test"
 
+    @property
+    def extra_authorize_data(self) -> dict:
```

Comment:
Is there a test to verify this is appended?

Reply:
The core tests coverage stayed the same (lines are hit) currently in the core tests for those properties (including this new one) is missing. 

The Toon tests actually verify them additionally.